### PR TITLE
Updated FAQ to remove AMA statements

### PR DESCRIPTION
**Pull request by paulgrimley**
Removed outdated AMA FAQ item and reworded some related statements to refer to deprecation of MMA being in the past

### CODE REVIEW

**Review by Springstone — APPROVED** [review]
LGTM

File: docs/wiki/FAQ.md [hunk 1]
```diff
@@ -155,15 +155,13 @@ The Management Group Names/IDs created via the ALZ Portal Accelerator Deployment
...
   - `<Resource Prefix (Root ID)>-sandbox`
 
-## Why hasn't Azure landing zones migrated to the Azure Monitor Agent yet?
+## Azure Monitor Agent
 
-**Update January 2024** We have been working on the removal of MMA from ALZ and the first step in the overall removal process is to update the ALZ Portal reference implementation (greenfield deployments) which has now been updated. Our next step is to work on the deployment to Terraform and Bicep reference implementations which requires significant investment to minimise impact to existing customers and providing clear guidance for the transition. For more details please see [Azure Monitor Agent Update](./ALZ-AMA-Update.md).
+### What if we are not ready to make the switch (from MMA) and migrate to AMA, right now?
 
-### What if we are not ready to make the switch and migrate, right now?
+Another good question. You will need to plan, and complete, the migration to the Azure Monitor Agent as soon as possible as the Log Analytics Agent has retired as [documented here.](https://azure.microsoft.com/updates/were-retiring-the-log-analytics-agent-in-azure-monitor-on-31-august-2024/)
```
Comment (Springstone) [inline docs/wiki/FAQ.md:162]:
```suggestion
Good question. You will need to plan, and complete, the migration to the Azure Monitor Agent as soon as possible as the Log Analytics Agent has retired as [documented here.](https://azure.microsoft.com/updates/were-retiring-the-log-analytics-agent-in-azure-monitor-on-31-august-2024/)
```

**Review by Springstone — CHANGES_REQUESTED** [review]

File: docs/wiki/FAQ.md (hunk 1 above)
Comment (arjenhuitema) [inline docs/wiki/FAQ.md:162]:
```suggestion
The log analytics agent (MMA) has retired as documented [here]( https://azure.microsoft.com/updates/were-retiring-the-log-analytics-agent-in-azure-monitor-on-31-august-2024/). Cloud ingestion services will gradually reduce support for MMA agents, which may result in compatibility issues over time. Ingestion for MMA will remain unchanged until February 1, 2025. You need to complete the migration to the Azure Monitor Agent before that date. 
```

**Review by arjenhuitema — CHANGES_REQUESTED** [review]
Thanks @paulgrimley,

Sacha's suggestion got me thinking. I made an additional suggestion; this rewrite includes the details shared in the [documentation](https://learn.microsoft.com/en-us/azure/azure-monitor/agents/azure-monitor-agent-migration) about what to expect. The suggested answer provides a clearer understanding of the impact and the necessary time frame.

**Comment by paulgrimley** [comment]
Happy with the suggestions, thanks @arjenhuitema @Springstone!

**Review by Springstone — APPROVED** [review]
LGTM

**Review by arjenhuitema — APPROVED** [review]
LGTM

**Pull request merged**